homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -21,7 +21,7 @@
     ATTR_ENTITY_ID, CONF_HOST, CONF_NAME, CONF_TOKEN, STATE_OFF, STATE_ON)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['python-mirobo==0.2.0']
+REQUIREMENTS = ['python-miio==0.2.0']
```

Comment:
You need to review the `mirobo` imports down there, not all are changed to `miio` imports.
Example: L92

Reply:
Yup, they are there for now as there has been no real releases of python-miio yet. I'm still working on some changes on python-miio I want to get it in before pushing this PR forwards. For now it's just for curious people to test it out (requires installing the master branch atm).